src/frontend/screens/Library/components/GamesList/index.tsx

```diff
@@ -26,8 +56,66 @@ const GamesList = ({
   onlyInstalled = false,
   isRecent = false
 }: Props): JSX.Element => {
-  const { gameUpdates } = useContext(ContextProvider)
+  const { gameUpdates, showNonAvailable } = useContext(ContextProvider)
   const { t } = useTranslation()
+  const [gameCards, setGameCards] = useState<JSX.Element[]>([])
+
+  useMemo(() => {
```

Comment:
sorry, I just realized the createGameCards stuff is async and useMemo is not intended for async memoization, sorry for suggesting that, useEffect is the right one here :see_no_evil: 

with sync content we could do `const gameCards = useMemo(() => ....)` instead of using useState+useEffect, but we can't here

Reply:
hmm, weird that it did not broke anything, maybe the status update could break using memo, not sure.
anyway, reverted to useEffect